README.md

```diff
@@ -50,6 +50,11 @@ A lot of the functionality of the `teal` framework derives from the following pa
 -   [`teal.logger`](https://insightsengineering.github.io/teal.logger/latest-tag/): standardizes logging within `teal` framework.
 -   [`teal.reporter`](https://insightsengineering.github.io/teal.reporter/latest-tag/): allows `teal` applications to generate reports.
 
+Additionally, a video has been prepared to enhance your teal learning experience.  
+Please click on the image below to watch it.
+
+[![A Complete Guide to Getting Started with teal](https://img.youtube.com/vi/N8ZamECICSI/0.jpg)](https://www.youtube.com/watch?v=N8ZamECICSI)
+
```

Comment:
```suggestion
Dive deeper into `teal` with our comprehensive video guide. Click the image below to start learning:

<p align="center">
  <a href="https://www.youtube.com/watch?v=N8ZamECICSI">
    <img src="https://img.youtube.com/vi/N8ZamECICSI/0.jpg" alt="A Complete Guide to Getting Started with teal">
  </a>
</p>

```

How about this. 
updated the text and center the image.

Reply:
Thanks @kartikeyakirar 
I like the wording of your sentence but I will keep the README without the html code.

Reply:
[c763b8c](https://github.com/insightsengineering/teal/pull/1206/commits/c763b8c34bd2389f6aaeda2bf49a72745fbedd09)